drivers/display/ssd16xx.c

```diff
@@ -700,6 +700,6 @@ static struct display_driver_api ssd16xx_driver_api = {
 
 
 DEVICE_DT_INST_DEFINE(0, ssd16xx_init, NULL,
-		    &ssd16xx_driver, NULL,
+		    &ssd16xx_driver, &ssd16xx_config,
```

Comment:
No, do not change it, driver is not broken, regarding confusing configuration see https://github.com/zephyrproject-rtos/zephyr/pull/39708

Reply:
I see, driver uses this weird pattern. Dropped for now (will eventually get done properly).